src/raylib/textures/Texture2D.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** Textures
+*/
+
+#include "Texture2D.hpp"
+
+namespace raylib
+{
+    namespace textures
+    {
+
+        Texture2D::Texture2D(Vector2 position, char *fileName)
+        {
+            _fileName = fileName;
+            _position = position;
+            _texture = loadTexture();
+        }
+
+        Texture2D::~Texture2D()
```

Comment:
You can use `~Texture2D() = default` in `Texture2D.hpp` instead of defining an empty destructor